CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# v1.8.0
+
+### `@liveblocks/node`
+
+- Add utilities to work with the `CommentBody` format from Comments:
+  - `getMentionIdsFromCommentBody(body)` - Get a list of all mention IDs from a
+    `CommentBody`.
+  - `stringifyCommentBody(body, options)` - Convert a `CommentBody` to a string,
+    either as plain text, HTML, or Markdown. It supports resolving mention IDs
+    similarly to `@liveblocks/react` and overriding each element to control the
+    formatting.
```

Comment:
May be nice to include a link to the documentation right in these release notes too.